bigtable/client/instance_admin.cc

```diff
@@ -108,6 +108,80 @@ google::bigtable::admin::v2::Instance InstanceAdmin::CreateInstanceImpl(
   return result;
 }
 
+std::future<google::bigtable::admin::v2::Cluster> InstanceAdmin::CreateCluster(
+    ClusterConfig instance_config, std::string const& instance_id,
+    std::string const& cluster_id) {
+  return std::async(std::launch::async, &InstanceAdmin::CreateClusterImpl, this,
+                    std::move(instance_config), instance_id, cluster_id);
+}
+
+google::bigtable::admin::v2::Cluster InstanceAdmin::CreateClusterImpl(
+    ClusterConfig const& cluster_config, std::string const& instance_id,
+    std::string const& cluster_id) {
```

Comment:
Please use strong types for `instance_id` and `cluster_id`.  I believe other functions can serve as examples.